addons/survey/static/src/js/survey_session_chart.js

```diff
@@ -209,13 +210,13 @@ publicWidget.registry.SurveySessionChart = publicWidget.Widget.extend({
                     const nbrCol = chart.data.labels.length;
                     const minRatio = 0.4;
                     // Numbers of maximum characters per line to print based on the number of columns and default ratio for the font size
-                    // Between 1 and 2 -> 25, 3 and 4 -> 20, 5 and 6 -> 15, ...
+                    // Between 1 and 2 -> 35, 3 and 4 -> 30, 5 and 6 -> 15, ...
```

Comment:
I believe it's now 30 for "5 and 6", right?

We could rephrase to:
`// We use 35 characters from 1 to 2 columns and then 30 characters for the rest.`

Reply:
Hello @awa-odoo, thanks for pointing it out, adapted the changes we're GTG :)